gulpfile.js

```diff
@@ -62,12 +62,23 @@ gulp.task('watch', function () {
 /**
  * Validate source JavaScript
  */
-gulp.task('jshint', function () {
+gulp.task('lint', function () {
```

Comment:
When linting, the user shouldn't know what linter they are using. Users should just invoke the command, and resolve conflicts.